student.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :name, :age
+
+  def initialize(name, age)
+    @name = name
+    @age = age
+  end
+
+  def can_use_services?
+    @age >= 18
+  end
+end
```

Comment:
- [ ] You created the `Person` class in the `person.rb` file. Kindly delete these blocks of codes so that your code is **_DRY (Don't Repeat Yourself)_**. If you want to get access to the Person class, all you have to do is require it in this file by doing `require_relative "person.rb"` at the top of the file. 👍 💻